spec/CordovaCheck.spec.js

```diff
@@ -31,12 +31,12 @@ describe('findProjectRoot method', function () {
         process.chdir(cwd);
     });
     function removeDir (someDirectory) {
-        shell.rm('-rf', someDirectory);
+        fs.removeSync(someDirectory);
     }
     it('Test 001 : should return false if it hits the home directory', function () {
         var somedir = path.join(home, 'somedir');
         removeDir(somedir);
-        shell.mkdir(somedir);
+        fs.ensureDirSync(somedir);
```

Comment:
In this particular instance, one could use `fs.emptyDirSync(somedir)` to replace the remove/create idiom